pkg/signature/kms/cliplugin/options.go

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cliplugin implements the plugin functionality.
+package cliplugin
+
+import (
+	"context"
+	"crypto"
+	"time"
+
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/sigstore/sigstore/pkg/signature/kms/cliplugin/common"
+)
+
+// packRPCOptions extracts properties of all of opts into struct ready for serializing.
+func packRPCOptions(opts []signature.RPCOption) *common.RPCOptions {
+	ctx := context.TODO()
```

Comment:
We should be opinionated on a default context, setting some reasonable timeouts. I think `context.Background()` should also be used rather than `TODO`.

Reply:
I agree, but the opinion should come from our primary caller cosign, which does set a 3minute timeout. 
Changed the contexts to Background.